tests/torch/test_compile_torch.py

```diff
@@ -1258,6 +1265,24 @@ def test_fancy_indexing_torch(model_object, default_configuration):
     compile_brevitas_qat_model(model, x, n_bits=4, configuration=default_configuration)
 
 
+@pytest.mark.parametrize(
+    "model_object",
+    [
+        pytest.param(MultiOutputModel),
+    ],
+)
+def test_multi_output(model_object, default_configuration):
+    """Test fancy indexing torch."""
+    model = model_object()
+    x = numpy.random.randint(0, 2, size=(100, 3, 10)).astype(numpy.float64)
+    y = numpy.random.randint(0, 2, size=(100, 3, 10)).astype(numpy.float64)
+    quantized_module = compile_torch_model(
+        model, (x, y), n_bits=4, configuration=default_configuration
+    )
+    quantized_module.forward(x[[0]], y[[0]])
+    quantized_module.forward(x[[0]], y[[0]], fhe="simulate")
```

Comment:
Could we check that we have indeed two output quantizers and that they are applied correctly?

Reply:
I can check that we have two quantizers but not sure how you want to assert that they are applied correctly

Reply:
yes I agree ! We should at least check that we actually have two outputs from this forward !

Reply:
This has been resolved but has this been done?